packages/handlefi/src/input-filters.ts

```diff
@@ -0,0 +1,190 @@
+import { type SwapActionParams } from '@rabbitholegg/questdk'
+import { getAddress, zeroAddress } from 'viem'
+import { buildV2PathQueryWithCase } from './utils'
+import {
+  PARASWAP_ABI,
+  V2_ROUTER_ABI,
+  HPSM2_ABI,
+  HLP_CURVE_V2_ABI,
+  HLP_BALANCER_ABI,
+  CURVE_FACTORY_ABI,
+} from './abi'
+import { WETH } from './constants'
+
+export function getParaSwapFilter(params: SwapActionParams) {
```

Comment:
This is super cool - this way we can just recycle the filter generators for a given protocol against aggregates! Good idea.